frontend/apps/monkvision/js/application.mjs

```diff
@@ -11,7 +11,7 @@ import {apimanager as apiman} from "/framework/js/apimanager.mjs";
 const init = async _ => {
 	window.APP_CONSTANTS = (await import ("./constants.mjs")).APP_CONSTANTS;
 	window.LOG = (await import ("/framework/js/log.mjs")).LOG;
-	window.APP_THEME = await $$.requireJSON(`${APP_CONSTANTS.APP_PATH}/conf/theme.json`);
+	monkshu_env.app_theme = await $$.requireJSON(`${APP_CONSTANTS.APP_PATH}/conf/theme.json`);
```

Comment:
This is wrong. The theme should be a URL param, which pageGenerator can directly ingest from the URL. We don't need to set an application variable and then use that later.